src/style.css

```
@@ -0,0 +1,108 @@
+:root {
+  height: 100%;
+  color: #545862;
+  font-size: 20px;
+  font-weight: 400;
+}
+
```

- [ ] [OPTIONAL] It would be nice if you could also add comments to your stylesheets. That way, it is easy to understand the styles that apply to different sections in the HTML. For example:
```suggestion
/* Body */
```